lib/guest_installation_and_configuration_base.pm

```diff
@@ -1355,9 +1382,15 @@ sub config_guest_installation_method {
     if ($self->{guest_installation_method} eq 'location') {
         $self->config_guest_installation_media;
         $self->{guest_installation_method_options} = "--location $self->{guest_installation_media}";
+    } elsif ($self->{guest_installation_method} eq 'import') {
+        $self->{guest_installation_method_options} = "--import ";
     }
-    $self->{guest_installation_method_options} = $self->{guest_installation_method_options} . ($self->{guest_installation_method_others} ne '' ? ",$self->{guest_installation_method_others}" : '') if ($self->{guest_installation_method_others} ne '');
-    $self->{guest_installation_method_options} = $self->{guest_installation_method_options} . ($self->{guest_installation_fine_grained} ne '' ? " --install $self->{guest_installation_fine_grained}" : '') if ($self->{guest_installation_fine_grained} ne '');
+
+    unless ($self->{guest_installation_method} eq 'import') {
+        $self->{guest_installation_method_options} = $self->{guest_installation_method_options} . ($self->{guest_installation_method_others} ne '' ? ",$self->{guest_installation_method_others}" : '') if ($self->{guest_installation_method_others} ne '');
+        $self->{guest_installation_method_options} = $self->{guest_installation_method_options} . ($self->{guest_installation_fine_grained} ne '' ? " --install $self->{guest_installation_fine_grained}" : '') if ($self->{guest_installation_fine_grained} ne '');
+    }
```

Comment:
```
    if ($self->{guest_installation_method} eq 'location') {
        $self->config_guest_installation_media;
        $self->{guest_installation_method_options} = "--location $self->{guest_installation_media}";
        $self->{guest_installation_method_options} = $self->{guest_installation_method_options} . ($self->{guest_installation_method_others} ne '' ? ",$self->{guest_installation_method_others}" : '') if ($self->{guest_installation_method_others} ne '');
        $self->{guest_installation_method_options} = $self->{guest_installation_method_options} . ($self->{guest_installation_fine_grained} ne '' ? " --install $self->{guest_installation_fine_grained}" : '') if ($self->{guest_installation_fine_grained} ne '');

    } elsif ($self->{guest_installation_method} eq 'import') {
        $self->{guest_installation_method_options} = "--import ";
    }
```
???

Reply:
What's your question?

Reply:
I think the new logic should be refined a bit as above.

Reply:
done